scripts/main.js

```diff
@@ -71,4 +71,62 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+const sales = departments.sales;
+const marketing = departments.marketing;
+const customerService = departments['customer-service'];
+let chosenJob = false;
+let departmentDescription = '';
+let jobsDescription = '';
+let errorMessage = '';
+
+let userInput = prompt('Over welke afdeling wil je meer informatie? \n(Marketing / Sales / Customer-Service)');
+
+switch(userInput.toLowerCase()) {
+    case 'marketing':
+        departmentDescription = marketing.description;
+        job = 'marketing';
+        break;
+    case 'sales':
+        departmentDescription = sales.description;
+        job = 'sales';
+        break;
+    case 'customer-service':
+        departmentDescription = customerService.description;
+        job = 'customer-service';
+        break;
+    default:
+        showError();
+}
+
+const jobs = departments[job].jobs;
+let jobString = '';
+for(let i=0;i <jobs.length;i++){
+    jobString += i +" "+jobs[i].title+"\n";
+}
+
+userInput = prompt("Je koos voor "+job+". Over welke functie wil je meer weten? " +
```

Comment:
Andere notatie:

userInput = prompt(`Je koos voor ${job}. Over welke functie wil je meer weten? Voer een getal tussen 0 en ${(jobs.length -1)} in. \n\n ${jobString}`);